core/connectors.js

```diff
@@ -200,7 +200,7 @@ define(function() {
 
 		{
 			label: 'Spotify',
-			matches: ['https://play.spotify.com/*'],
+			matches: ['https://player.spotify.com/*'],
```

Comment:
So, this would appear to be a problem - Spotify seem to be splitting this change geographically. I guess this PR is for the new interface in the US(?), which hasn't been rolled out globally. We may have to keep both connectors around for the foreseeable future.

To clarify:
1: The changes to the player aren't being observed in the play.spotify.com player in the UK.
2: player.spotify.com redirects to play.spotify.com in the UK at least, probably elsewhere too.

Otherwise, :+1:, looks good.